ServerCore/Pages/Threads/PuzzleThreads.cshtml.cs

```diff
@@ -0,0 +1,69 @@
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Identity;
+using Microsoft.AspNetCore.Mvc;
+using ServerCore.DataModel;
+using ServerCore.Helpers;
+using ServerCore.ModelBases;
+using System.Collections.Generic;
+using System;
+
+namespace ServerCore.Pages.Threads
+{
+    [AllowAnonymous]
+    public class PuzzleThreadsModel : EventSpecificPageModel
+    {
+        public const int PreviewTextLength = 100;
+
+        public PuzzleThreadsModel(PuzzleServerContext serverContext, UserManager<IdentityUser> userManager) : base(serverContext, userManager)
+        {
+        }
+
+        /// <summary>
+        /// The team of the current user. 
+        /// This value will only be filled in the play route and if the player actually has a team.
+        /// </summary>
+        public Team Team { get; set; }
+
+        /// <summary>
```

Comment:
Use case to think about: During an event, Game Control is going to want to easily filter this down to unclaimed messages only. Also, filter to specific puzzles.

Also, ideally GC will be able to see this page update live without needing to refresh. Not something that needs to be tackled in this first PR.

Reply:
I am putting all requests here
https://github.com/PuzzleServer/mainpuzzleserver/issues/969